modules/sr_site/manifests/discord_gated_entry.pp

```diff
@@ -0,0 +1,70 @@
+class sr_site::discord_gated_entry (
+  $git_root,
+  $root_dir,
+) {
+  user { 'discord':
+    ensure  => present,
+    comment => 'Discord bot service user',
+    shell   => '/sbin/nologin',
+    gid     => 'users',
+  }
+
+  vcsrepo { $root_dir:
+    ensure    => present,
+    provider  => git,
+    source    => 'https://github.com/srobo/discord-gated-entry',
+    revision  => 'origin/main',
+    owner     => 'discord',
+    group     => 'users',
+    require   => User['discord'],
+  }
+
+  $env_file = "${root_dir}/.env"
+  $discord_gated_entry_token = hiera('discord_gated_entry_token')
```

Comment:
This appears unused here, can we push the variable lookup into the template?

Reply:
Sure, how?

Reply:
I'm pretty sure you just put `hiera('discord_gated_entry_token')` where you had the variable access? Have a grep around for `hiera`. I'm pretty sure we do this elsewhere.

Reply:
I didn't see anything which matched that, and the internet wasn't much help.

Reply:
Huh, I was sure we did that somewhere. :shrug: